src/components/manager.ts

```diff
@@ -87,6 +85,27 @@ export class Manager {
         return path.normalize(out).split(path.sep).join('/')
     }
 
+    /**
+     * @returns the indicator if the root file
+     */
+    getRootIndictor() {
```

Comment:
Typo. Also make this a proper [Typescript getter](https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/).